build/DependencyVersions.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <CLI_SharedFrameworkVersion>2.0.0</CLI_SharedFrameworkVersion>
+    <CLI_SharedFrameworkVersion>2.1.0-preview2-25616-02</CLI_SharedFrameworkVersion>
```

Comment:
@nguerrera @eerhardt do you know why we haven't gotten an update to 2.1 bits in the CLI since this?  /cc @ahsonkhan

Reply:
See https://github.com/dotnet/cli/pull/7601.  Maestro was broke.

Reply:
Can you give a rough eta on when we can expect a new CLI with newer 2.1 shared framework version? Please cc me on the PR that updates the CLI_SharedFrameworkVersion. Tyvm.